hs-ucan/package.yaml

```diff
@@ -158,6 +159,8 @@ dependencies:
 
 library:
   source-dirs: library
+  pkg-config-dependencies:
+    - libsecp256k1
```

Comment:
I added this line to get it to build on the Github actions, but doesn't seem to change anything. The `libsecp256k1` package has this too.